src/python/pants/engine/internals/build_files.py

```diff
@@ -283,6 +317,39 @@ def _address_spec_to_globs(address_mapper: AddressMapper, address_specs: Address
     return PathGlobs(globs=(*patterns, *(f"!{p}" for p in address_mapper.build_ignore_patterns)))
 
 
+def error_on_imports(build_file_content, filepath, behavior):
```

Comment:
Type hints appreciated if you know them. I think `str` for the first two and `BuildFileImportsBehavior` for the last one? Returns `-> None`.